docling/models/code_formula_model.py

```diff
@@ -233,9 +233,9 @@ def __call__(
 
         labels: List[str] = []
         images: List[Union[Image.Image, np.ndarray]] = []
-        elements: List[TextItem] = []
+        elements: List[Union[TextItem | CodeItem]] = []
         for el in element_batch:
-            assert isinstance(el.item, TextItem)
+            assert isinstance(el.item, TextItem) or isinstance(el.item, CodeItem)
```

Comment:
This is necessary to work with the docling-core `CodeItem` change to be based on `FloatingItem`. The change will break prior docling versions working with the new docling-core.